store/router.go

```diff
@@ -54,9 +54,21 @@ func (r *Router) Get(ctx context.Context, key []byte, cm commitments.CommitmentM
 			r.log.Debug("Retrieving data from memstore")
 			return r.mem.Get(ctx, key)
 		}
-
-		r.log.Debug("Retrieving data from eigenda")
-		return r.eigenda.Get(ctx, key)
+		if r.s3 != nil && r.s3.cfg.Backup {
```

Comment:
yeah thinking about this - since the blob retrieved from S3 can be verified (certified) would this elevate blob reads from the public node? Might be a good feature if it's not sacrificing security.

Reply:
if not mistaken this feature is provided to ensure resiliency for blob reads in the event that it can't be read from DA.
CC - @bxue-l2